portal-api/src/v1/controllers/utilisation-report-service/utilisation-report-upload.controller.js

```diff
@@ -1,161 +1,80 @@
+const getReportForPeriod = async (bankId, reportPeriod) => {
+  const reportsForPeriod = await api.getUtilisationReports(bankId, {
     reportPeriod,
   });
-};
-
-/**
- * Sends notification email to bank payment officer team that a utilisation report has been
- * received and return the payment officer team email address.
- * @param {string} reportPeriod - period for which the report covers as a string, eg. June 2023
- * @param {string} bankId - the bank ID
- * @param {Date} submittedDate - the date the report was submitted
- * @param {string} submittedBy - the name of the user who submitted the report as a string
- * @returns {Promise} returns object with payment officer email or an error
- */
-const sendEmailToBankPaymentOfficerTeam = async (reportPeriod, bankId, submittedDate, user) => {
-  try {
-    const reportSubmittedBy = `${user.firstname} ${user.surname}`;
-    const { teamName, emails } = await getPaymentOfficerTeamDetailsFromBank(bankId);
-    const formattedSubmittedDate = formatDateForEmail(submittedDate);
 
-    await Promise.all(
-      emails.map((email) =>
-        sendEmail(EMAIL_TEMPLATE_IDS.UTILISATION_REPORT_CONFIRMATION, email, {
-          recipient: teamName,
-          reportPeriod,
-          reportSubmittedBy,
-          reportSubmittedDate: formattedSubmittedDate,
-        })),
+  if (reportsForPeriod.length !== 1) {
+    throw new Error(
+      `Expected 1 report but found ${
+        reportsForPeriod.length
+      } with bank ID ${bankId} and report period '${getFormattedReportPeriodWithLongMonth(reportPeriod)}'`,
     );
-    return { paymentOfficerEmails: emails };
-  } catch (error) {
-    console.error('Unable to get payment officer team details and send email %o', error);
-    return { status: error?.code || 500, data: 'Failed to get payment officer team details and send email' };
   }
-};
 
-/**
- * Saves file to Azure in utilisation-reports ShareClient, returns the file storage info
- * @param {object} file
- * @param {string} bankId - bank id as a string
- * @returns {Promise<object>} - azure storage details with folder & file name, full path & url.
- */
-const saveFileToAzure = async (file, bankId) => {
-  try {
-    console.info(`Attempting to save utilisation report to Azure for bank: ${bankId}`);
-    const { originalname, buffer } = file;
-
-    const fileInfo = await uploadFile({
-      fileshare: FILESHARES.UTILISATION_REPORTS,
-      folder: bankId,
-      filename: formatFilenameForSharepoint(originalname),
-      buffer,
-      allowOverwrite: true,
-    });
-
-    if (!fileInfo || fileInfo.error) {
-      throw new Error(`Failed to save utilisation report to Azure - ${fileInfo?.error?.message ?? 'cause unknown'}`);
-    }
-
-    console.info(`Successfully saved utilisation report to Azure for bank: ${bankId}`);
-    return fileInfo;
-  } catch (error) {
-    console.error('Failed to save utilisation report to Azure ', error);
-    throw error;
-  }
-};
+  return reportsForPeriod[0];
+}
```

Comment:
Missing semi-colon I think (now that the prettier PR has been merged, it should be possible to run `npm run prettier:fix` to fix any and all of these formatting things)